servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImplMain.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2017 Yahoo Holdings Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.athenz.zms;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
+import com.yahoo.athenz.auth.util.Crypto;
+
+public class ZMSImplMain {
+    private static final String AUDITREF = "zms_standalone";
```

Comment:
We should set the audit reference to something like - "Server bootstrap: instance private key registration"